advanced/folder-caseSensitiveFS.rst

```diff
@@ -11,8 +11,8 @@ handling.
 With ``caseSensitiveFS`` set to  ``false`` (the default setting)
 Syncthing's case sensitivity safety checks are enabled. 
 Syncthing will then attempt to detect and prevent case-only file
-name collisions that can occur on case insensitive systems such as Windows
-and macOS, or other systems with case insensitive file systems.
+name collisions that can occur on case insensitive systems such as Windows,
+macOS, and Android.
```

Comment:
How about something like this?

```
name collisions that can occur on case insensitive operating systems such as
Windows, macOS, iOS, and Android, or other systems with case insensitive file
systems.
```

I've changed the first "systems" to "operating systems" and also added iOS to the mix. I think these are the big four.

Reply:
I updated the PR to use your proposal.